.github/workflows/maven.yaml

```diff
@@ -7,12 +7,18 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Install
-        run: |
-          sudo apt update
-          sudo apt install maven
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
       - name: Package
         env:
           MAVEN_OPTS: -Dmaven.test.skip=true -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
         run: |
+          mvn -v
```

Comment:
Is the version output for logging purposes?

Reply:
Yes, exactly. I can see in the logs what the maven and java versions are this way and can cross reference any changes with previous builds/runs.